controllers/organization_validator.go

```diff
@@ -72,12 +73,14 @@ func (v *OrganizationValidator) handle(ctx context.Context, req admission.Reques
 		}
 	}
 
-	orgClusters := capiv1alpha3.ClusterList{}
+	orgClusters := capiv1alpha2.ClusterList{}
 	{
 		err := v.List(ctx, &orgClusters, &client.ListOptions{
 			LabelSelector: orgClustersSelector,
 		})
-		if err != nil {
+		if apimeta.IsNoMatchError(err) {
+			// If the CRD is not in place, just ignore the error.
```

Comment:
This helps on KVM, since this error would prevent org deletion on those installations, even through the REST API.